workflow/energy_rating_index.rb

```diff
@@ -519,6 +556,29 @@ def get_eec_dhws(hpxml_doc)
     else
       value_adj = 1.0
     end
+
+    ## Combi system requires recalculating ef
+    if value.nil?
+      combi_boiler_afue = 1.0
```

Comment:
Seems like we want this initialized to nil and not 1.0 so that, if the boiler afue is not found below, it generates an error rather than silently using an arbitrary boiler afue. Also, please move this closer to the loop where boiler afue is obtained.